ha/ut/helper.c

```diff
@@ -45,10 +44,11 @@ M0_INTERNAL void m0_ha_ut_rpc_ctx_init(struct m0_ha_ut_rpc_ctx *ctx)
 	const uint32_t tms_nr      = 1;
 	const uint32_t bufs_nr     =
 		m0_rpc_bufs_nr(M0_NET_TM_RECV_QUEUE_DEF_LEN, tms_nr);
-	const char    *ep          = "0@lo:12345:42:100";
-	int            rc;
+	const char         *ep          = "0@lo:12345:42:100";
+	int                 rc;
```

Comment:
Please restore the alignment of these two variables.

Reply:
incorporated